manifests/custom_hook.pp

```diff
@@ -1,26 +1,44 @@
-# @summary Manage custom hook files within a GitLab project. Custom hooks can be created as a pre-receive, post-receive, or update hook. It's possible to create different custom hook types for the same project - one each for pre-receive, post-receive and update.
+# @summary
+#     Manage custom hook files within a GitLab project.
+#
+# Custom hooks can be created as a pre-receive, post-receive, or update hook.
+# Only one of each is currently supported by this module.
```

Comment:
please undo this. Check the REFERENCE.md above. It now doesn't render properly anymore.

Reply:
Weird, it looked OK to me in VSCode and on Github. Undone.